### ISSUE TITLE
Apparent Error in the track_query implementation

### ISSUE THREAD

**Issue by hawaiiansun** [GXNU-ZhongLab/ODTrack#11]
Hi,

Thanks for the great work on this project!

While the official released checkpoints perform as expected (achieving 72.9 AUC on LaSOT compared to the claimed 73.1), I'm encountering a significant discrepancy when attempting to reproduce the model training. My training runs consistently result in a final AUC of 60.6 on LaSOT, a 12.5% difference from the reported results.

After investigating the codebase, I believe the issue might stem from the implementation of the ODTrack class, specifically the forward function ([Link](https://github.com/GXNU-ZhongLab/ODTrack/blob/9f286cfa6146bf0aec9ca1be4340d68e083cd11f/lib/models/odtrack/odtrack.py#L41)). While it seems to function correctly during inference due to the tracker being re-initialized for each sequence, during training, `self.track_query` retains values from the previous batch. This behavior appears incorrect and could explain the performance gap.

Could you please review this section of the code and potentially rerun the training experiments to verify the results?

This issue seems related to the previously raised concern in #5.

Any insights or updates would be greatly appreciated.

**Comment by hawaiiansun** [comment]
@Azong-HQU @Leonhard-Euler-ai

**Comment by sunqimin** [comment]
Hello, I have a question about ’self.track_query‘, 
’if self.backbone.add_cls_token:
self.track_query = (x[:, :self.token_len].clone()).detach() # stop grad  (B, N, C)‘
Isn't the condition false, and what does that have to do with ’track_query‘？And do you know which part of the temporal token in the paper corresponds to the code？I would appreciate it if you could answer my questions！

**Comment by 2609780105** [comment]
您好，这个问题解决了吗？
